src/openrct2/object/ImageTable.cpp

```diff
@@ -174,19 +175,19 @@ std::vector<std::unique_ptr<ImageTable::RequiredImage>> ImageTable::ParseImages(
     auto raw = Json::GetString(el["format"]) == "raw";
     auto keepPalette = Json::GetString(el["palette"]) == "keep";
     auto zoomOffset = Json::GetNumber<int32_t>(el["zoom"]);
+    auto spritePalette = DefaultPalette;
```

Comment:
why are you copying the palette here?

Reply:
Aborted change to this function to use custom palettes, I will do that in another PR, I forgot to remove this line.